src/VisualStudio/IntegrationTest/New.IntegrationTests/AbstractEditorTest.cs

```diff
@@ -60,7 +61,7 @@ public override async Task InitializeAsync()
         protected async Task ClearEditorAsync(CancellationToken cancellationToken)
             => await SetUpEditorAsync("$$", cancellationToken);
 
-        protected async Task SetUpEditorAsync(string markupCode, CancellationToken cancellationToken)
+        protected async Task SetUpEditorAsync(string markupCode, CancellationToken cancellationToken, string contentType = ContentTypeNames.RoslynContentType)
```

Comment:
See other comment: the code needing a content type seems wrong, so you can revert this.